crates/flowctl/src/controlplane.rs

```diff
@@ -50,27 +66,98 @@ pub(crate) fn new_client(config: &Config) -> anyhow::Result<Client> {
     }
 }
 
-pub fn configure_new_access_token(ctx: &mut CliContext, token: String) -> anyhow::Result<()> {
+pub async fn configure_new_access_token(ctx: &mut CliContext, token: String) -> anyhow::Result<()> {
```

Comment:
Just nit picking, but having different types of tokens makes this code a bit hard to read without specific and consistent names for `refresh_token` vs `access_token`. Maybe rename the `token` parameter to `access_token`?